features/step_definitions/assertion_steps.rb

```diff
@@ -23,6 +22,11 @@
   end
 end
 
+And(/^"([^"]*)" should be set in "([^"]*)"$/) do |status, list|
+  dropdown = page.find("##{list}")
+  selected_option = dropdown.find('option[selected]').text
+  expect(selected_option).to eql status
+end
```

Comment:
I don't think it does. It returned Pending until I got the logic on the view working (which yes, is to be moved/refactored)